apps/storybook-react/tailwind.config.js

```diff
@@ -1,12 +1,12 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Need to figure out workspace package imports in the apps/ workspace currently we are running into this error 

```Error:   1:36  error  Unable to resolve path to module '@metamask/design-system-tailwind-preset'  import/no-unresolved```